ieeg_motorMiller2007/participants.tsv

```diff
@@ -0,0 +1,20 @@
+name	sex	age
+sub-bp	n/a	n/a
+sub-ca	n/a	n/a
+sub-cc	n/a	n/a
+sub-de	n/a	n/a
+sub-fp	n/a	n/a
+sub-gc	n/a	n/a
+sub-gf	n/a	n/a
+sub-hh	n/a	n/a
+sub-hl	n/a	n/a
+sub-jc	n/a	n/a
+sub-jf	n/a	n/a
+sub-jm	n/a	n/a
+sub-jp	n/a	n/a
+sub-jt	n/a	n/a
+sub-rh	n/a	n/a
+sub-rr	n/a	n/a
+sub-ug	n/a	n/a
+sub-wc	n/a	n/a
+sub-zt	n/a	n/a
```

Comment:
This file is optional so there is not reason to include it if it's full of empty values.